### DISCUSSION TITLE
Is it possible to adjust the color of the lightbox image effect?

### DISCUSSION THREAD

**Discussion by davidbeckonline** [McShelby/hugo-theme-relearn#837] [Q&A] [answered]
Hello. I am wondering whether the color of the lightbox image effect can be adjusted.
On my website, I am using a lot of blue. But it is a different blue than the one the lightbox is using in order to highlight an image when hovering over it.

I have searched trough the theme css, but it does not seem to be an adjustable parameter in there.

**Comment by McShelby** [comment]
Do you mean this?

![grafik](https://github.com/McShelby/hugo-theme-relearn/assets/11670870/ee420998-3ffb-4e0b-aac5-3e1be7501233)

Or that?

![grafik](https://github.com/McShelby/hugo-theme-relearn/assets/11670870/2ce9cbbf-a7ec-4307-b41c-e85365f756c9)

The second was never to be intended. I will remove that border in a minute. See #838

**Reply by davidbeckonline** [reply]
Thank you for your feedback.

I meant both, because I think that both are part of the lightning, aren't they?

If the second scenario (image opened in lightbox) is removed, then I would still like to adjust the colour of the first scenario (hover over image), if this is possible.

**Reply by McShelby** [reply]
It was intended to use `MAIN-LINK-color` and `MAIN-LINK-HOVER-color` for this (as it is basically a link) - which obviously isn't.

I'll change this later today.

Would this be sufficient for your case?

**Reply by davidbeckonline** [reply]
Yes, that would be enough and awesome.

**Reply by McShelby** [reply]
This is harder as initially thought. While this already works as intended for Chrome, Firefox requires more tweaking.

**Reply by davidbeckonline** [reply]
Thank you for looking into this @McShelby .

**Reply by McShelby** [reply, marked as answer]
Should now all be fixed with #838

**Reply by davidbeckonline** [reply]
Confirming that `MAIN-LINK-color` and `MAIN-LINK-HOVER-color`are considered for the lightbox after upgrading the Theme to the latest release (see pegel-konstanz.de).

Thank you very  much for your support.
